sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -247,6 +247,10 @@ src_configure() {
 	multilib-minimal_src_configure
 }
 
+get_rootprefix() {
+	usex split-usr "${EPREFIX:-/}" "${EPREFIX}/usr"
+}
```

Comment:
The code itself looks good.
Ideally we should minimize difference with Gentoo.
I believe you would send the changes to upstream in the future. :-)

Reply:
I think that systemd ebuild is the most customized forked ebuild that we have in overlay. So I think we would need to do a bit of work on this ebuild to make some part of it upstreamable either to gentoo or maybe even to systemd.